### ISSUE TITLE
use `Model` instances instead of `MappingType` instances

### ISSUE THREAD

**Issue by Rizziepit** [universalcore/springboard#24]
Is there a reason we are not using the `MappingType.to_object` method to convert to model instances, e.g. https://github.com/universalcore/springboard/blob/develop/springboard/views.py#L63?

I want to prevent future issues when schemas change.

**Comment by Rizziepit** [comment]
@miltontony @smn is this issue valid?

**Comment by smn** [comment]
No, the only reason that's not used is because when this was writing the only option available was to either hit Git or work with whatever ES returned.

**Comment by smn** [comment]
So, yes, this issue is valid :)
